### PR TITLE
Jchen peds 528 sftp uploader

### PR DESCRIPTION
**Pull request by JiaweiChen110**
As a user of the PEDSnet pipeline, I would like to be able to upload the final .tar.gz payload to CHOP using a workflow script, so that this process is automated.

### CODE REVIEW

File: src/main/wdl/common.wdl [hunk 1]
```diff
@@ -144,4 +144,39 @@ task S3Upload {
...
+        File source
+        String dockerImage = "wdl-kit:1.3.0"
+    }
+
+    command <<<
+        python3 <<CODE
+        import pysftp
+        cnopts = pysftp.CnOpts()
+        cnopts.hostkeys = None
+        with pysftp.Connection('~{sftpPort}', username='~{sftpUser}', password='~{sftpPassword}', cnopts= cnopts) as sftp:
```
Comment (JiaweiChen110) [inline src/main/wdl/common.wdl]: sftpAddr

File: Makefile [hunk 2]
```diff
@@ -11,6 +11,7 @@ docker:
 install: 
 	pip install -r requirements.txt
 	pip3 install target/dist/stanford-wdl-kit-$(VERSION)/dist/stanford-wdl-kit-$(VERSION).tar.gz
+	pip install pysftp
```
Comment (jmesterh) [inline Makefile:14]: this needs to go into requirements.txt so that the Dockerfile will pick it up also

Comment (jmesterh) [inline Makefile]: shouldn't be needed since it's already in requirements.txt

File: src/main/wdl/common.wdl [hunk 3]
```diff
@@ -144,4 +144,39 @@ task S3Upload {
...
+        cnopts.hostkeys = None
+        with pysftp.Connection('~{sftpAddr}', username='~{sftpUser}', password='~{sftpPassword}', cnopts= cnopts) as sftp:
+            sftp.cd('~{sftpLocation}')           # temporarily chdir to sftpLocation
+            sftp.put('~{source}')  	# upload file to source on remote
+            sftp.close()
+        CODE
+    >>>
+
+    output {
+        String dir = read_string(stdout())
```
Comment (jmesterh) [inline src/main/wdl/common.wdl]: change `dir` to `log` or `out`?

**Review by jmesterh — APPROVED** [review]

**Pull request merged**